tests/unit/test_issues.py

```diff
@@ -9,12 +9,16 @@
 
 import unittest
 import requests
+import json
 
 from mock import patch
+from mock import ANY
```

Comment:
Note to self about [ANY](https://docs.python.org/3/library/unittest.mock.html#any).